prov/rxm/src/rxm_conn.c

```diff
@@ -606,8 +606,11 @@ int rxm_msg_eq_progress(struct rxm_ep *rxm_ep)
 			break;
 		}
 		ret = rxm_conn_handle_event(rxm_ep, entry);
-		if (ret)
+		if (ret) {
+			FI_DBG(&rxm_prov, FI_LOG_EP_CTRL,
+			       "invalid connection handle event: %d\n", ret);
```

Comment:
A recommendation: 

```"invalid connection handle event: %d (%s)\n", ret, fi_strerror(ret))```

Or something similar perhaps. I think it would be nice to have the string error, but unnecessary.

Reply:
Thanks.